### PR TITLE
chore(lint): Update tools, fix lint violations

### PR DESCRIPTION
**Pull request by antonblock**
### Proposed Change

- Update versions in `internal/tools/go.mod` to be at least as up to date as the collector
  - Downgrade the `swag` tool to when it didn't panic
- Fix new lint violations by adding missing package-level comments

##### Checklist
- [ ] Changes are tested
- [ ] CI has passed

### CODE REVIEW

**Review by andykellr — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -42,7 +42,6 @@ install-tools: install-git-hooks
 	cd $(TOOLS_MOD_DIR) && go install honnef.co/go/tools/cmd/staticcheck
 	cd $(TOOLS_MOD_DIR) && go install github.com/client9/misspell/cmd/misspell
 	cd $(TOOLS_MOD_DIR) && go install github.com/ory/go-acc
-	cd $(TOOLS_MOD_DIR) && go install github.com/vektra/mockery/v2@v2.20.0
```
Comment (andykellr) [inline Makefile:45]: Why is this removed?

Comment (antonblock) [reply Makefile:45]: Doh, only meant to remove the pinned version. Just added it back

**Review by jsirianni — APPROVED** [review]
